### PR TITLE
Small README fixes

### PR DESCRIPTION
**Pull request by mayaCostantini**
## This introduces a breaking change

-  No

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -16,30 +16,30 @@ This is a Promotheus Metrics exporter for Thoth.
...
+4. Set the metric variable value from ``metrics.py`` in the method that you define. More, on that here on `prometheus documentation <https://github.com/prometheus/client_python#gauge>`__.
 5. Finally if you have a new class added in jobs, add it to the `init.py <https://github.com/thoth-station/metrics-exporter/blob/master/thoth/metrics_exporter/jobs/__init__.py>`__.
 
 Conventions
 ===========
 
 For any binary metrics the convention used is:
 
-- 0 all good
+- `0` all good
```
Comment (pacospace) [inline README.rst:40]: double ``?

**Review by pacospace — COMMENTED** [review]
One minor comment otherwise lgtm! Thanks!

File: README.rst (hunk 1 above)
Comment (mayaCostantini) [reply README.rst:40]: Right, thanks :100:

**Review by pacospace — APPROVED** [review]
Thanks 💯 🚀

**Pull request merged**